docs/runbook.md

```diff
@@ -0,0 +1,266 @@
+Runbook
+=======
+
+For regular operations and maintenance of the Router back-end of the Console
+
+## Once Per New Server
+
+General installation instructions are available on the Helium documentation
+website.
+
+> Start here:
+[Run a Network Server](https://docs.helium.com/use-the-network/run-a-network-server/)
+which goes from `git clone` through to various initial `docker` commands
+such as loading an initial snapshot and testing with a real or virtual device.
+
+Syntax of **Bash** shell is used below.
+
+Create shell aliases.
+
+Router:
+
+```bash
+alias router='docker exec -it helium_router router'
+alias wallet='~/helium-wallet-rs/target/release/helium-wallet'
+```
+
+Console:
+
+```bash
+alias console_down='docker-compose down'
+alias console_psql='docker exec -it helium_postgres psql -U postgres'
+alias console_restart='console_down && console_up'
+alias console_up='docker-compose build && docker-compose up -d && docker-compose logs -f'
+alias console_update='docker-compose build --pull && console_restart'
+```
+
+With the above set of aliases defined and available in your current shell,
+confirm:
+
+```bash
+router --help
+```
+
+## Stop/Start
+
+On production servers, Console versus Router run on different hosts to
+isolate and distribute workloads.
+
+Because staging has far less traffic, everything runs on one host.
+
+```bash
+docker-compose down
```

Comment:
worth noting here or somewhere early that docker-compose commands are meant to be run from specific directories?  it prints a fairly obvious error, but a short warning may speed recovery for unfamiliar folks

Reply:
Good catch!  Fixed in 64b4652